README.md

```diff
@@ -1,13 +1,22 @@
 # Project Auth
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was built by Yennie Leek and Sandra Wallén as part of the Technigo bootcamp. The main task was to build an application where a user can sign up/log in and see some "secret" content if they are loged in.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We started with setting up the endpoints and requests in backend, one POST to sign up as a user, another to log in as an existing user, a PATCH to add additional information to the user account when they are logged in and one POST and one GET to post and get thoughts. We have only implemented functionality for getting the thoughts in frontend, as it is now posts can only be made from postman or other similar service.
+
+We have a problem in the sign up functionality, if the sign up wasn't successful for any reason the system continues reading and sending "success: true" even though it wasn't, the expected result would be that the system exits and catches the error, but it doesn't.
+
+There is alot of functionality and tweeks that we would have wanted to implement to this project but we didn't have much time to work on this project unfortunately.
+
+### If we had more time we would have
+- Shown in the profile page if the user had previously added the additional account information
+- Added functionality to post a thought in the feed
+- Improved all styling
+- Handled error messages user friendly in the frontend.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+View frontend live at https://cranky-brown-a574ee.netlify.app/
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```

![image](https://user-images.githubusercontent.com/50926159/120920775-d1c06700-c6c0-11eb-8717-324c4bf65dea.png)